### ISSUE TITLE
Issue in replicating the result 

### ISSUE THREAD

**Issue by sfarkya** [bknyaz/videos#2]
Hello Dear Boris, 

I have been trying to replicate the result of your classification on UCF101 using VGG16, and I am not quite able to able to achieve the accuracy you have mentioned (between 73-75.5 for various combinations), my training is overfitting the data very quickly., getting around the accuracy of 50% while the training accuracy is above 90%. So, I just wanted to confirm the hyperparameter you used to perform experiments and other settings. 

P.S. I wrote code separately in Tensorflow and following your setting. 

So, if you remember anything please let me know if there's any difference, it would be really helpful  

**Training Data** 
All the frames are used from all the training set given in split 1 of the dataset. 

**Data Augmentation:** 
 1. Randomly cropping an original image of size (320, 240, 3) to size (224, 224, 3). 
Only cropping is used (I included some more to rectify the overfitting)

**Transfer learning**
ImageNet weights are used for the whole network (including convolutional and dense layers)  and various combination of finetuning is done for experimentation as mentioned  [here](https://github.com/bknyaz/videos#results) 
Q. I just wanted to confirm if the Imagenet weights you used were just for Conv layers OR dense layers as well and the classifier for 101 labels was added by removing the ImageNet classifier. 

**Regularizer**
I think you used L2 weight decay and for some experiments, dropout was used. Batch Normalization was never used. 

**Training Duration**
Training is done just for 15K iterations (meaning 15k randomly shuffled batches were passed). It seems very few epochs of training is required. 

**Learning rate**
Anywhere between .001 to .0001 as starting, No decay was used. 

**Batch size = 64**
**Optimizer:** Momentum (with momentum=.9)

**Testing** 
Every other third frame from the test data was used to report the accuracy. 

I think that's all the settings I could gather. Please let me know if I am missing something Or If am not doing anything correctly. 

Thank you in advance for your help!. 

Best, 
Saurabh
